content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md

```diff
@@ -495,7 +495,7 @@ kube-controller-manager [flags]
 <td colspan="2">--feature-gates colonSeparatedMultimapStringString</td>
```

Comment:
Don't change this. This is a generated file and will be updated after the release is cut.

Reply:
Fixed.